src/ccache.cpp

```diff
@@ -146,6 +146,8 @@ Options for scripting or debugging:
     -k, --get-config KEY       print the value of configuration key KEY
         --hash-file PATH       print the hash (160 bit BLAKE3) of the file at
                                PATH
+        --log-stats            print statistics counters from the stats log
```

Comment:
I would appreciate if  documentation of the new command line option and the configuration option could be added to the manual. Also, some test cases would be awesome!